routes/error-tracker.js

```diff
@@ -61,12 +61,9 @@ function handler(req, res, params) {
 
   let errorType = 'default';
 
-  if (singlePassType) {
-    if (singlePassType === 'sp') {
-      errorType += '-sp';
-    } else if (singlePassType === 'mp') {
-      errorType += '-mp';
-    }
+  const singlePassTypes = ['sp', 'mp', 'esm'];
+  if (singlePassTypes.includes(singlePassType)) {
```

Comment:
I'm not sure the check is necessary. If there's a `signlePassType`, let's append it.

Reply:
done.

Reply:
i'll do the same thing on the runtime